docs/i18n/ja/docusaurus-plugin-content-docs/current/getting-started/demo/without-docker.md

```diff
@@ -12,13 +12,24 @@ import TabItem from '@theme/TabItem';
 
 > Dockerコンテナなしで、実行ファイルとスクリプトでデモを実行します。
 
-:::info Context
-以下のコマンドはすべて、プロジェクトリポジトリの `demo` フォルダから実行されるものとして書かれています。したがって、何かをする前にリポジトリをクローンして `cd demo` を実行して該当ディレクトリにポインターを合わせてください。
+## 準備
+
+スコープに「cardano-node」、「hydra-node」、および「hydra-tui」実行可能ファイルがあることを確認してください。 次のいずれかを実行できます
+
+ - `nix-shell demo` を使用するか、
```

Comment:
```suggestion
 - `nix-shell -A demo` を使用するか、
```

Reply:
This changed now. It's not an attribute anymore, but a separate .nix file.

Reply:
Ah sorry.